src/ophys_etl/modules/mesoscope_splitting/zstack_splitter.py

```diff
@@ -56,6 +62,10 @@ def __init__(self, tiff_path_list: List[pathlib.Path]):
                 raise RuntimeError("Could not find discretePlaneMode==0 for "
                                    f"{tiff_path}")
 
+            if this_roi not in roi_to_path:
+                roi_to_path[this_roi] = []
```

Comment:
This is not necessary if you use a `defaultdict(list)` when initializing `roi_to_path` on L48.

Reply:
I'll incorporate this into my future work. I should probably start writing python like it's *not* 2011....